src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpCodeActions.cs

```diff
@@ -215,12 +215,7 @@ class C
             VisualStudio.Editor.Verify.CodeAction(
                 "Use expression body for properties",
                 applyFix: true,
-                fixAllScope: FixAllScope.Project,
-                blockUntilComplete: false);
-
-            var expectedTitle = "Preview Changes - Fix all occurrences";
```

Comment:
Why don't we verify this anymore?

Reply:
Two reasons:

1. Testing the Preview Changes window is not a core purpose of this test.
2. Testing the Preview Changes window required passing `blockUntilComplete: false`  to the previous call, which results in the operation not getting joined and thus some types of failures would go unobserved.

Modifying the code to bypass the preview window improved reliability of this critical functionality test, and we can work on another Preview Changes test later.

Reply:
Can you log a bug tracking adding a Preview Changes test? (If only from the perspective of tracking that we no longer have this coverage, where we did before).

Reply:
Yep, filed #27806